packer/jenkins-agent-al2-arm64.json

```diff
@@ -1,6 +1,6 @@
 {
   "variables": {
-    "name-base":"Jenkins-Agent-AL2-ARM64-JDK21",
+    "name-base":"Jenkins-Agent-AL2-ARM64",
```

Comment:
What were name-base used for? Just want to make sure we are not breaking any existing funtionality

Reply:
We dont need to add those into the AMI name itself.
As these are just templates to run scripts.
The scripts itself already contain the JDK version it needs to install.

Reply:
I believe we are using AMI ID in our configs so this change wont affect us or any community user using jenkins specs right?

Reply:
Yes, It wont.

Reply:
The only thing matters is the AMI id here.